integration_test/third_party_apps_data/applications/varnish/centos_rhel/install

```diff
@@ -1,8 +1,25 @@
 set -e
 
-# Centos Stream / RHEL 8: Varnish 6.0.8
+source /etc/os-release
+if [ "$ID" = centos ] && [ "$VERSION_ID" = 7 ]; then
+    sudo yum install -y epel-release
+    sudo tee /etc/yum.repos.d/varnishcache_varnish60lts.repo > /dev/null <<-EOF
+[varnishcache_varnish60lts]
+name=varnishcache_varnish60lts
+baseurl=https://packagecloud.io/varnishcache/varnish60lts/el/${VERSION_ID%%.*}/$(arch)
+repo_gpgcheck=0
+gpgcheck=0
+enabled=1
+gpgkey=https://packagecloud.io/varnishcache/varnish60lts/gpgkey
+sslverify=1
+sslcacert=/etc/pki/tls/certs/ca-bundle.crt
+metadata_expire=300
+EOF
+fi
 
+# Centos Stream / RHEL 8: Varnish 6.0.8
```

Comment:
i think delete this comment, since it's misleading -- these commands run on more than just those versions, and it doesn't necessarily install that that version of varnish

Reply:
Good point, missed that cleanup after copying the existing code around.